src/Impostor.Api/Net/Messages/Announcements/MessageHello.cs

```diff
@@ -0,0 +1,15 @@
+using Impostor.Api.Innersloth;
+
+namespace Impostor.Api.Net.Messages.Announcements
+{
+    public class MessageHello
+    {
+        public static void Deserialize(IMessageReader reader, out int announcementVersion, out int id, out Language language)
+        {
+            reader.ReadByte(); // TODO the hec is this
```

Comment:
https://twitter.com/Sanae6k/status/1338631404758843392

Reply:
are you sure its a short? isn't it one byte? (first byte should be there because hazel)

Reply:
both bytes are there because hazel
are you 100% sure it's actually the hazel version and not a stray byte?